account_bank_statement_import_adyen/tests/test_import_adyen.py

```diff
@@ -0,0 +1,62 @@
+from odoo import tests
+from .test_import_file import TestStatementFile
+
+
+@tests.tagged('post_install', 'at_install')
```

Comment:
defaults are fine

Reply:
defaults? can you write what are the defaults?

Reply:
https://github.com/odoo/odoo/blob/12.0/odoo/tests/common.py#L196

Reply:
Oh, to be clear: the thing about defaults is that you don't have to add them here.